stumpy/stump.py

```diff
@@ -454,6 +479,9 @@ def stump(T_A, m, T_B=None, ignore_trivial=True):
         start, T_A, T_B, m, excl_zone, M_T, Σ_T, ignore_trivial
     )
 
+    # Only after calculating the first profile we can remove all nan values from T_B
+    T_B[np.isnan(T_B)] = 0
```

Comment:
Append the following comment to the end of the line `  # Remove all nan values from T_B only after first profile is calculated`